blog/_posts/2016-08-15-hello-from-team-xyz.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Hello from Team XYZ
+layout: post
+created_at: Mon August 15 2016
+permalink: blog/2016-08-10-hello-from-team-xyz
```

Comment:
can you change the permalink here to the correct date? (it should be `blog/2016-08-15` instead of `blog/2016/08/10`)